chasm/src/main/java/org/quiltmc/chasm/internal/TransformerSorter.java

```diff
@@ -100,11 +126,20 @@ public static List<List<Transformer>> sort(Collection<Transformer> transformers)
                 }
             } while (checkAgain);
 
+            // If no roundInfos were added, then dependencies did not change. Therefore, no roundInfos will be added
+            // next loop either, so we're in an infinite loop.
+            if (roundInfo.isEmpty()) {
```

Comment:
I would prefer if your comment said what the code is doing, rather than why it's correct.
Sometihng along the lines of "Abort if no more transformers could be added (because of a dependency cycle)"